BCnEnc.Net/Decoder/BcDecoder.cs

```diff
@@ -43,26 +43,103 @@ public class BcDecoder
 		/// <param name="pixelHeight">The pixelHeight of the image.</param>
 		/// <param name="token">The cancellation token for this asynchronous operation.</param>
 		/// <returns>The awaitable operation to retrieve the decoded Rgba32 image.</returns>
-		public Task<Image<Rgba32>> DecodeRawAsync(Stream inputStream, CompressionFormat format, int pixelWidth, int pixelHeight, CancellationToken token = default)
+		public Task<Memory<ColorRgba32>> DecodeRawAsync(Stream inputStream, CompressionFormat format, int pixelWidth, int pixelHeight, CancellationToken token = default)
```

Comment:
Keep in mind that there are a lot of APIs out there that take in arrays, but do not take in `Memory<T>`.  If a user wanted to use such an API from other libraries they would not be able to since they can't convert the memory to an array without copy.

Reply:
I changed the normal decoding to return an array now. I do see the merit that just a normal array is more flexible for the user. There's also an alternative overload for each method that returns a Memory2D, which is more in line with the encoder methods.